presto-product-tests/bin/run_on_docker.sh

```diff
@@ -78,6 +100,14 @@ PRODUCT_TESTS_ROOT="${SCRIPT_DIR}/.."
 PROJECT_ROOT="${PRODUCT_TESTS_ROOT}/.."
 DOCKER_COMPOSE_LOCATION="${PRODUCT_TESTS_ROOT}/conf/docker/${ENVIRONMENT}/docker-compose.yml"
 
+SINGLE_NODE_DOCKER_COMPOSE_LOCATION="${PRODUCT_TESTS_ROOT}/conf/docker/singlenode/docker-compose.yml"
+DISTRIBUTED_DOCKER_COMPOSE_LOCATION="${PRODUCT_TESTS_ROOT}/conf/docker/distributed/docker-compose.yml"
```

Comment:
for each new test configuration we would have to add line here. Until there are 2 or 3 it is fine, but if there are more, then separate script should be created for each configuration.